pylint/utils/utils.py

```diff
@@ -17,12 +17,32 @@
 import sys
 import textwrap
 import tokenize
+from typing import TYPE_CHECKING, Any, List, Optional, Pattern, Tuple, Union, overload
 
 from astroid import Module, modutils
 
 from pylint.constants import PY_EXTS
 
+if TYPE_CHECKING:
+    from typing import Literal
```

Comment:
This will fail if someone tries to run mypy on Python 3.7
Instead do something like this:
```py
if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```